css/style.css

```diff
@@ -120,28 +130,23 @@ h3 {
 	font-size: 24px;
 }
 
-input:focus {
-	border: 2px solid #16b898;
+footer {
+	padding: 20px 0;
 }
 
-input [type="submit"]
-
 @media (max-width: 480px) {
-    {
-        top: 50%; /* IMPORTANT */
-        left: 50%; /* IMPORTANT */
+	
+/* This is not doing anything */
+/*	.thing {
```

Comment:
Not sure what you were trying to do here but these styles were not attached to anything. Just commented out incase you needed them.